src/components/environment-badge.tsx

```diff
@@ -2,7 +2,7 @@ import { __ } from '@wordpress/i18n';
 import { Badge } from 'src/components/badge';
 import { cx } from 'src/lib/cx';
 
-export type EnvironmentType = 'production' | 'staging';
+export type EnvironmentType = 'production' | 'staging' | 'sandbox';
```

Comment:
@ivan-ottinger , Let's remove the export of this type since it is not used anywhere else.

```suggestion
type EnvironmentType = 'production' | 'staging' | 'sandbox';
```

Reply:
Good catch! Thanks!